apps/docs/src/app/docs/cron-jobs/page.md

```diff
@@ -1,31 +0,0 @@
----
```

Comment:
We are deleting the file instead of rename it to `background-jobs`.

Reply:
done